twitter/friendships_test.go

```diff
@@ -45,3 +45,31 @@ func TestFriendshipService_Destroy(t *testing.T) {
 	expected := &User{ID: 12345, Name: "Doug Williams"}
 	assert.Equal(t, expected, user)
 }
+
+func TestFriendshipService_Show(t *testing.T) {
+	httpClient, mux, server := testServer()
+	defer server.Close()
+
+	mux.HandleFunc("/1.1/friendships/show.json", func(w http.ResponseWriter, r *http.Request) {
+		assertMethod(t, "GET", r)
+		assertQuery(t, map[string]string{"source_screen_name": "foo", "target_screen_name": "bar"}, r)
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprintf(w, `{ "relationship": { "source": { "can_dm": false, "blocking": null, "muting": null, "id_str": "8649302", "all_replies": null, "want_retweets": null, "id": 8649302, "marked_spam": null, "screen_name": "foo", "following": false, "followed_by": false, "notifications_enabled": null }, "target": { "id_str": "12148", "id": 12148, "screen_name": "bar", "following": false, "followed_by": false } } }`)
```

Comment:
Several of these fields aren't actually tested against. You can strip them if they're not a part of the deserialization sanity test.